src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -1171,6 +1171,24 @@ fn generic_simd_intrinsic(
     );
     let arg_tys = sig.inputs();
 
+    if name == "simd_select_bitmask" {
+        let in_ty = arg_tys[0];
+        let m_len = match in_ty.sty {
+            ty::Int(i) => i.bit_width().unwrap(),
+            ty::Uint(i) => i.bit_width().unwrap(),
```

Comment:
I think this will crash with usize/isize? Probably not a big issue but could be avoided by computing the layout.

Reply:
It will indeed! That was semi-intentional, but I'll leave a comment in case a future reader is curious

Reply:
Might be better to properly error here?

Reply:
Hm perhaps yeah, but this is all internal anyway so I don't think we really need to go too far out of our way...